docs/rfc/0002_kubectl_rw.md

```diff
@@ -0,0 +1,49 @@
+|                    |                                      |
+| -------            |--------------------------------------|
+| Feature            | kubectl-rw: a kind of kubectl plugin |
```

Comment:
Will `rw` be too ambiguous to be used as the plugin name? What is your opinion @twocode?

Reply:
but if use `risingwave`, it's too long. Or use `rise`? What is your opinion @twocode

Reply:
Personally I think it's okay to use it now. 😁

Reply:
> but if use `risingwave`, it's too long. Or use `rise`? What is your opinion @twocode

> Personally I think it's okay to use it now. 😁

Yeah, makes sense. Though I still prefer a meaningful name, like `risingwave`, `risectl` or other names that mean managing RisingWaves in Kubernetes, but it's fine to name it `rw` for now.